*** testsBoth/A-11-06-M-ok.out	Sun Oct 30 22:57:52 2022
--- testsBoth/A-11-06-M-ok.outhyp	Sat Nov 12 13:15:21 2022
***************
*** 29,34 ****
--- 29,35 ----
  Escolha uma opção: 9
  Número do terminal: 969005
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
+ O terminal '969005' não pode receber comunicações VOICE
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 37,43 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 38,45 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
***************
*** 76,82 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 7
! Valores para o terminal '969007':0 (pagamentos),  10 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 78,84 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 7
! Valores para o terminal 969007':0 (pagamentos),  16 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
