*** testsBoth/A-12-05-M-ok.out	Sun Oct 30 22:57:52 2022
--- testsBoth/A-12-05-M-ok.outhyp	Sat Nov 12 13:15:22 2022
***************
*** 29,35 ****
  Escolha uma opção: 9
  Número do terminal: 969008
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
! O terminal de destino ('969008') está em silêncio.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 29,35 ----
  Escolha uma opção: 9
  Número do terminal: 969008
  Tipo de comunicação a iniciar (VOICE, VIDEO): VOICE
! O terminal '969008' não pode receber comunicações VOICE
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
