*** testsBoth/A-15-03-M-ok.out	Sun Oct 30 22:57:54 2022
--- testsBoth/A-15-03-M-ok.outhyp	Sat Nov 12 13:15:27 2022
***************
*** 102,109 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
! TEXT|1|969001|969007|100|200|FINISHED
! TEXT|2|969011|969014|102|204|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 102,109 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
! TEXT|0|969001|969007|units|200.0|FINISHED
! TEXT|0|969011|969014|units|204.0|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
