*** testsBoth/A-15-04-M-ok.out	Sun Oct 30 22:57:55 2022
--- testsBoth/A-15-04-M-ok.outhyp	Sat Nov 12 18:23:36 2022
***************
*** 49,55 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 10
! Custo da comunicação: 200 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 49,55 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 10
! Custo da comunicação: 0 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 58,65 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 58,64 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
***************
*** 98,104 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 40
! Custo da comunicação: 800 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 97,103 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 40
! Custo da comunicação: 0 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 107,114 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 106,112 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
