*** testsBoth/A-15-05-M-ok.out	Sun Oct 30 22:57:55 2022
--- testsBoth/A-15-05-M-ok.outhyp	Sat Nov 12 18:23:36 2022
***************
*** 42,48 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 30
! Custo da comunicação: 900 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,48 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 30
! Custo da comunicação: 0 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 51,58 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
--- 51,57 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 0
***************
*** 81,87 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
! VIDEO|1|969011|969014|30|900|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 80,86 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: 1
! VIDEO|3|969011|969014|0|0|ONGOING
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
