*** testsBoth/A-18-12-M-ok.out	Sun Oct 30 22:57:57 2022
--- testsBoth/A-18-12-M-ok.outhyp	Sat Nov 12 18:23:44 2022
***************
*** 151,157 ****
  0 - Sair
  Escolha uma opção: 1
  Identificador do cliente: cli001
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 151,157 ----
  0 - Sair
  Escolha uma opção: 1
  Identificador do cliente: cli001
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|3|0|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
***************
*** 163,169 ****
  Escolha uma opção: 1
  Identificador do cliente: cli002
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
- O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 163,168 ----
***************
*** 175,181 ****
  Escolha uma opção: 1
  Identificador do cliente: cli003
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
- O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 174,179 ----
