*** tests/A-13-07-M-ok.out	Tue Nov  1 19:13:11 2022
--- tests/A-13-07-M-ok.outhyp	Mon Nov 14 17:03:43 2022
***************
*** 47,53 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 47,53 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 59,64 ****
--- 59,77 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
+ Escolha uma opção: O terminal já estava ligado.
+ OPERAÇÕES DO TERMINAL
+ 1 - Ligar
+ 2 - Colocar no silêncio
+ 3 - Desligar
+ 4 - Adicionar amigo
+ 5 - Retirar amigo
+ 6 - Pagamento
+ 7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
+ 11 - Mostrar comunicação em curso
+ 0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
***************
*** 80,86 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|260|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 93,99 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|260
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
***************
*** 162,168 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|260|260
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 175,181 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|520
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
***************
*** 205,211 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 218,224 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 217,222 ****
--- 230,247 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
+ Escolha uma opção: OPERAÇÕES DO TERMINAL
+ 1 - Ligar
+ 2 - Colocar no silêncio
+ 3 - Desligar
+ 4 - Adicionar amigo
+ 5 - Retirar amigo
+ 6 - Pagamento
+ 7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
+ 11 - Mostrar comunicação em curso
+ 0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
***************
*** 238,244 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|520|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 263,269 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|520
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
