*** tests/A-15-09-M-ok.out	Fri Nov  4 10:18:51 2022
--- tests/A-15-09-M-ok.outhyp	Mon Nov 14 17:03:58 2022
***************
*** 75,81 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 100 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 75,81 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 200 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 111,117 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: VOICE|1|969014|969011|10|200|FINISHED
! VOICE|2|969011|969014|10|100|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 111,117 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: VOICE|1|969014|969011|10|200|FINISHED
! VOICE|2|969011|969014|10|200|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
