*** tests/A-18-12-M-ok.out	Wed Oct 26 17:32:07 2022
--- tests/A-18-12-M-ok.outhyp	Mon Nov 14 17:04:05 2022
***************
*** 137,143 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: Identificador do cliente: CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
- O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 137,142 ----
***************
*** 147,153 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: Identificador do cliente: CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
- O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 146,151 ----
