*** tests/A-22-08-M-ok.out	Mon Oct 31 11:49:52 2022
--- tests/A-22-08-M-ok.outhyp	Mon Nov 14 17:04:21 2022
***************
*** 128,135 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: VOICE|1|969006|969001|1|20|FINISHED
! CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
--- 128,134 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
***************
*** 138,143 ****
--- 137,143 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
