*** tests/A-22-06-M-ok.out	Mon Oct 31 11:15:14 2022
--- tests/A-22-06-M-ok.outhyp	Mon Nov 14 17:04:20 2022
***************
*** 163,173 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: VOICE|1|969014|969001|1|20|FINISHED
! VOICE|2|969007|969011|1|20|FINISHED
! TEXT|3|969007|969011|6|10|FINISHED
! VOICE|4|969015|969013|1|20|FINISHED
! CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
--- 163,169 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
***************
*** 176,181 ****
--- 172,178 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
