*** tests/A-24-06-M-ok.out	Tue Nov  1 16:21:44 2022
--- tests/A-24-06-M-ok.outhyp	Mon Nov 14 17:04:25 2022
***************
*** 116,122 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 116,134 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
